lib/util.js

```diff
@@ -423,3 +423,13 @@ function compareByGeneratedPositionsInflated(mappingA, mappingB) {
   return strcmp(mappingA.name, mappingB.name);
 }
 exports.compareByGeneratedPositionsInflated = compareByGeneratedPositionsInflated;
+
+/**
+ * Strip any JSON XSSI avoidance prefix from the string (as documented
+ * in the source maps specification), and then parse the string as
+ * JSON.
+ */
+function parseSourceMapInput(str) {
+  return JSON.parse(str.replace(/^\)]}'(?:[^\n]*?\n)/, ''));
```

Comment:
Some simplifications:
* You don't need the grouping :)
* you don't need the ungreedy variant `*?` thanks to your character class

so I think `/^\)]}'[^\n]*\n/` works fine and is simpler :)

Reply:
Yes, duh.  Thanks.